platform/iphone/in_app_store.mm

```diff
@@ -30,36 +30,16 @@
 
 #ifdef STOREKIT_ENABLED
 
-#include "in_app_store.h"
-
-extern "C" {
+#import "in_app_store.h"
 #import <Foundation/Foundation.h>
 #import <StoreKit/StoreKit.h>
-};
+
+InAppStore *InAppStore::instance = NULL;
 
 bool auto_finish_transactions = true;
 NSMutableDictionary *pending_transactions = [NSMutableDictionary dictionary];
 
-@interface SKProduct (LocalizedPrice)
-@property(nonatomic, readonly) NSString *localizedPrice;
-@end
-
-//----------------------------------//
-// SKProduct extension
-//----------------------------------//
-@implementation SKProduct (LocalizedPrice)
-- (NSString *)localizedPrice {
-	NSNumberFormatter *numberFormatter = [[NSNumberFormatter alloc] init];
-	[numberFormatter setFormatterBehavior:NSNumberFormatterBehavior10_4];
-	[numberFormatter setNumberStyle:NSNumberFormatterCurrencyStyle];
-	[numberFormatter setLocale:self.priceLocale];
-	NSString *formattedString = [numberFormatter stringFromNumber:self.price];
-	[numberFormatter release];
-	return formattedString;
-}
-@end
-
-InAppStore *InAppStore::instance = NULL;
+static Dictionary TransactionDictionary(SKPaymentTransaction *transaction);
```

Comment:
Helper function to construct `Variant` `Dictionary`s that we'll pass back to Godot.